admin-dev/themes/new-theme/js/app/components/checkbox.vue

```diff
@@ -0,0 +1,87 @@
+<!--**
+     * 2007-2019 PrestaShop and Contributors
+     *
+     * NOTICE OF LICENSE
+     *
+     * This source file is subject to the Open Software License (OSL 3.0)
+     * that is bundled with this package in the file LICENSE.txt.
+     * It is also available through the world-wide-web at this URL:
+     * https://opensource.org/licenses/OSL-3.0
+     * If you did not receive a copy of the license and are unable to
+     * obtain it through the world-wide-web, please send an email
+     * to license@prestashop.com so we can send you a copy immediately.
+     *
+     * DISCLAIMER
+     *
+     * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+     * versions in the future. If you wish to customize PrestaShop for your
+     * needs please refer to https://www.prestashop.com for more information.
+     *
+     * @author    PrestaShop SA <contact@prestashop.com>
+     * @copyright 2007-2019 PrestaShop SA and Contributors
+     * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+     * International Registered Trademark & Property of PrestaShop SA
+     *-->
+<template>
+  <div class="md-checkbox md-checkbox-inline">
+    <label>
+      <input
+        v-if="Array.isArray(checked)"
+        type="checkbox"
+        :checked="checked.includes(value)"
+        :class="classes"
+        :disabled="disabled"
+        @change="change"
+      />
+      <input
+        v-else
+        type="checkbox"
+        :checked="checked"
+        :class="classes"
+        :disabled="disabled"
+        @change="$emit('input', $event.target.checked)"
+      />
+
+      <i class="md-checkbox-control"></i>
```

Comment:
Even cooler thing here would be to add 
```html
<slot>
<!-- Fallback content -->
<i class="md-checkbox-control"></i>
</slot>
```
this will allow people to choose whenever to use bootstrap 4 checkbox style or not. Also this will allow people to add some kind of text to the checkbox as if I recall correctly :+1:

Reply:
Good idea. I made it as simple as possible. I'll change it!